atlas/getting-started/deploy-from-existing-repo.mdx

```diff
@@ -97,6 +100,7 @@ If you're using the Faust.js Starter Kit or an existing Faust.js app repository,
 
 ![First app deployed](./assets/img/first-app-deployed.png)
 
+<a name="env-var"></a>
```

Comment:
@dgallardox What's the purpose of this empty link tag? Was this left in on accident, maybe?